controllers/dbconfig.php

```diff
@@ -1,13 +1,14 @@
 <?php
     class DbconfigController extends ControllerBase {
         protected $method = 'create';
-        protected function dbInit() {}
+        // protected function dbInit() {} // deprecated dunno why this was here. 
 
         public function create( $user, $pass, $dbname ) {
             $entries = compact( 'user', 'pass', 'dbname' );
             $entries = [ 'db' => $entries ];
             try {
                 updateConfig( $entries, $this->environment );
+                $this->runMigrations();

```

Comment:
I don't think this script was ever meant to run migrations, only to write a config file. Perhaps it should take you to a different controller to do that, or have a more generic installation script? I agree that it would be good to do that, but this is not the appropriate place to put it.